x-pack/plugins/observability_ai_assistant/server/service/kb_service/index.ts

```diff
@@ -37,9 +37,16 @@ export class KnowledgeBaseService {
 
   constructor(private readonly dependencies: Dependencies) {
     this.ensureTaskScheduled();
+    this.status().then((response) => {
+      this.hasSetup = response.ready;
+
+      if (response.ready) {
+        this.processQueue();
+      }
+    });
   }
 
-  private ensureTaskScheduled() {
+  private ensureTaskScheduled = () => {
```

Comment:
The method notation caused the method to not be callable. Arrow notation caused it to be callable.